release-notes/v2.1.0.md

```diff
@@ -0,0 +1,36 @@
+# HPE CSI Driver for Kubernetes 2.1.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.1.0
+
+| **Version:** |2.1.0|
+|--------------|-----|
+| **Revision:** | Tuesday January 10, 2022 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.1.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+
+The following table lists the resolved issues for HPE Alletra 9000 and Primera/3PAR CSP for HPE CSI Driver release 2.1.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v2.1.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+
+
+The following table lists the known issues for HPE Alletra 9000 and Primera/3PAR CSP for HPE CSI Driver release 2.1.0
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-2330|csi.k8s|Discrepancy in volume size between array and master. | When volume size is not provided in multiple of 256 then array round it off to nearby 256 multiple value but on the node level it's size get not reflected. |None|
```

Comment:
Description: When volume size is not provided in multiples of 256 Mib, the array provisions the volume with a size that is rounded off to the nearest 256 multiple, however the k8s shows the original (unrounded) volume size, hence the discrepancy.

Reply:
modified the same content